centreon/packages/ui/src/Dialog/index.tsx

```diff
@@ -71,6 +78,12 @@ const Dialog = ({
 }: Props): JSX.Element => {
   const { classes, cx } = useStyles({ contentWidth });
 
+  const dataTestIdConfirmButton =
+    dataTestId?.dataTestIdConfirmButton ?? labelConfirm;
+
+  const dataTestIdCanceledButton =
+    dataTestId?.dataTestIdCanceledButton ?? labelCancel;
+
```

Comment:
the label can change according to the language, therefore it's not a constant and should not be used

it would work for the tests on the ci, but not an option for product analytics (which is the goal of this PR if I understood well)

Reply:
for labels i didn t used the translation t(labelConfirm) so it s a constant 🤔

Reply:
ok, but one can pass the labels as props to the `Dialog` component, so those labels can be using `t()`